recipes/buttons/callbar_button/callbar_button.vue

```diff
@@ -132,6 +132,16 @@ export default {
       default: 'xl',
       validator: size => VALID_WIDTH_SIZE.includes(size),
     },
+
+    /**
+     * The fill and outline of the button associated with its visual importance.
+     * @values clear, outlined, primary
+     */
+    importance: {
+      type: String,
+      default: '',
+    },
```

Comment:
ah yeah just noticed this.. make sure to add this new prop to `callbar_button_default.story.vue` or it won't work in the storybook example

Reply:
Importance is also not showing up in the storybook controls.

Reply:
actually something looks messed up with the config on this story... taking a look

Reply:
> actually something looks messed up with the config on this story... taking a look

Ha, found it... Can you change the import in `callbar_button.stories.vue`

from: `import { DtRecipeCallbarButton, VALID_WIDTH_SIZE } from './callbar_button';`
to: `import DtRecipeCallbarButton, { VALID_WIDTH_SIZE } from './callbar_button';`

that should fix the issue with the props not showing up.